### ISSUE TITLE
iOS: Cursor not changed to black in white and sepia theme

### ISSUE THREAD

**Issue by Eikansh** [omnivore-app/omnivore#2144]
Hi,

Thanks for the application. It is working great for most part. 

The cursor to select text don't get changed to dark when switched to light themes on iOS. This make sit hard to follow the text I am going to highlight. The time and other symbols in the status bar of the iOS are in light mode too. 

I am using iPhone 12 with system wide appearance set to dark.

**Comment by jacksonh** [comment]
Thanks, this might be one thats harder to fix than expected because of the way those colours are applied. Will investigate more.

**Comment by jacksonh** [comment]
Fixed in 1.27 (should be available on TestFlight shortly).

**Comment by jacksonh** [comment]
Will close this one now as its in TestFlight, we will probably tweak the colours a little and then release to the AppStore.
